model/mmwave-sidelink-spectrum-phy.cc

```diff
@@ -271,16 +282,20 @@ MmWaveSidelinkSpectrumPhy::StartRxData (Ptr<MmWaveSidelinkSpectrumSignalParamete
       // If there are other intereferent devices that transmit in the same slot, the current
       // device simply does not consider the signal and goes on with the transmission. The code does not raise any errors since
       // otherwise we are not able to study scenarios where interference could be an issue.
-      break;
+      //break;
     case RX_CTRL:
-      NS_FATAL_ERROR ("Cannot receive control in data period");
-      break;
+      //NS_FATAL_ERROR ("Cannot receive control in data period");
+      //break;
     case RX_DATA:
       // If this device is in the RX_DATA state and another call to StartRx is
       // triggered, it means that multiple concurrent signals are being received.
       // In this case, we assume that the device will synchronize with the first
       // received signal, while the other will act as interferers
       m_interferenceData->AddSignal (params->psd, params->duration);
+      
+      //COMMENT BY ALESSANDRO: in case TX, RX_CTRL, RX_DATA comments talk about interferences. Thus I removed the break(s) and put a 
```

Comment:
Il caso RX_INTERFERENCE si riferisce al caso in cui l'unico segnale che ricevi è interferenza (un nome alernativo potrebbe essere "rx not useful signal"). Non serviva cambiare questa parte

Reply:
Per chiarire ulteriormente: se tu stai ricevendo dei dati utili (i.e., che sono diretti effettivamente a te) ma ricevi anche interferenza, questa deve essere valutata come tale rispetto al segnale utile (vedi definizione di SNR, per valutare se un pacchetto o bit è correttamente ricevuto).